lib/internal/crypto/hkdf.js

```diff
@@ -110,7 +110,15 @@ function hkdf(hash, key, salt, info, length, callback) {
 
   validateFunction(callback, 'callback');
 
-  const job = new HKDFJob(kCryptoJobAsync, hash, key, salt, info, length);
+  const job = new KDFJob(kCryptoJobAsync,
+    "HKDF",
+    {
+      digest: hash,
+      key: key.export({ format: 'buffer' }),
```

Comment:
I am not sure about this. OpenSSL expects an octet string and manipulating a key object in C++ sounded complicated